tests/test_api.py

```diff
@@ -17,7 +17,7 @@ def bliss_client(cwd):
 
 @pytest.fixture(scope="class")
```

Comment:
Pytest has started taking 35 minutes to finish, up from 1 or 2 minutes before `test_api.py` was added. It'd be good to keep the total runtime of pytest below 2 or 3 minutes. Main culprits:
```
992.28s setup    tests/test_api.py::TestApi::test_generate
826.95s call     tests/test_api.py::TestApi::test_train_on_cached_data
168.83s call     tests/test_api.py::TestApi::test_generate
```

Reply:
It may be that we can't test it so thoroughly, which is fine.

Reply:
Will address in #795.